ddspipe_core/include/ddspipe_core/types/dds/Payload.hpp

```diff
@@ -17,12 +17,12 @@
 
 #include <fastdds/dds/core/policy/QosPolicies.hpp>
```

Comment:
Maybe you can sort ascending the lines of this block. Same in the rest of files.